submission/submission-webapp/src/main/resources/application.yml

```diff
@@ -23,10 +23,10 @@ spring:
       max-request-size: 20GB
       location: ${app.tempDirPath}
 
-logging.file: logs/application_logs.txt
 logging:
   file:
     max-history: 7
+    name: logs/application_logs.txt
```

Comment:
did you test this?

Reply:
Yes, it was deployed in dev and the logs were properly generated.